### PR TITLE
hear me out...

### PR DESCRIPTION
**Pull request by PikachuGabe**
multiple dragon blades since they suck anyways

### CODE REVIEW

**Comment by flerouwu** [comment]
Should I be scared?

**Comment by PikachuGabe** [comment]
all i did is create a function that summons the dragonblade item but without all tag work being done on the altar(and added the script to the dragon death function as a "only execute if altar is active" function)

**Comment by flerouwu** [comment]
Is this a threat?

**Comment by flerouwu** [comment]
Honestly, I just realised Cores would be good for something like this. Would let pika test it himself and we could have a preset with some V+ plugins.

@Cooperly

**Comment by Cooperly** [comment]
> Honestly, I just realised Cores would be good for something like this. Would let pika test it himself and we could have a preset with some V+ plugins.
> 
> @Cooperly

counter point: giving pika their own mini server on damascus that hibernates if unused so it doesn't waste resources

cores could be useful for creating fresh instances automatically on every commit but i don't know how to implement it in a non janky manner and this can just be done with a small server on the panel and uploading everything manually and full wiping the world files (excluding datapacks fold) if you absolutely need a clean slate

TL;DR: i'm stupid and i don't know how to implement it :3 
(and we don't even have the necessary plugin and custom wings fork to feasibly be able to do cores yet so)

**Comment by PikachuGabe** [comment]
> Honestly, I just realised Cores would be good for something like this. Would let pika test it himself and we could have a preset with some V+ plugins.
> 
> @Cooperly

I could probably test this on my pc by building it rq
